### PR TITLE
Fix no site logo empty link issue for ticket 60922

### PR DESCRIPTION
**Pull request by itzmekhokan**
Fix site logo missing empty link render for SEO and accessibility purpose.

Trac ticket: [#60922](https://core.trac.wordpress.org/ticket/60922)

---
**This Pull Request is for code review only. Please keep all other discussion in the Trac ticket. Do not merge this Pull Request. See [GitHub Pull Requests for Code Review](https://make.wordpress.org/core/handbook/contribute/git/github-pull-requests-for-code-review/) in the Core Handbook for more details.**

### CODE REVIEW

**Review by joedolson — CHANGES_REQUESTED** [review]
This looks good, but we should probably add a similar check in `has_custom_logo()` so that the conditional is consistent with what we output.

**Comment by itzmekhokan** [comment]
> This looks good, but we should probably add a similar check in `has_custom_logo()` so that the conditional is consistent with what we output.

@joedolson I have update the PR.

File: src/wp-includes/general-template.php [hunk 1]
```diff
@@ -1029,7 +1029,7 @@ function has_custom_logo( $blog_id = 0 ) {
 		restore_current_blog();
 	}
 
-	return (bool) $custom_logo_id;
+	return (bool) wp_attachment_is_image( $custom_logo_id );
```
Comment (joedolson) [inline src/wp-includes/general-template.php]: `wp_attachment_is_image()` only returns boolean, so it isn't necessary to cast this.

**Review by joedolson — CHANGES_REQUESTED** [review]
Minor change.

File: src/wp-includes/general-template.php (hunk 1 above)
Comment (itzmekhokan) [reply src/wp-includes/general-template.php]: @joedolson updated the line.

**Review by joedolson — APPROVED** [review]
I made a change to fix the failing tests; approving this with those changes.

**Review by joedolson — APPROVED** [review]

**Comment by joedolson** [comment]
Fixed in r58213

**Pull request closed without merging**